build-scripts/ubuntu-1804/postinst

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Automatically added from template:
+if which py3compile >/dev/null 2>&1; then
+	py3compile -O -p {package_name} /usr/local/lib/python3.5/dist-packages/
```

Comment:
On ubuntu 18 we will plan to use python3.6 . In this case, we need to take into account dist-packages location for python3.6